src/main/java/ru/promo/consul_plan/config/SecurityProperties.java

```diff
@@ -0,0 +1,20 @@
+package ru.promo.consul_plan.config;
```

Comment:
```suggestion
package ru.promo.consul_plan.config.properties;
```
обычно помещаем сюда